core/deployment/src/main/java/io/quarkus/deployment/steps/NativeImageAutoFeatureStep.java

```diff
@@ -205,7 +208,26 @@ public void write(String s, byte[] bytes) {
         }
 
         if (!resourceBundles.isEmpty()) {
-            ResultHandle locClass = overallCatch.loadClass(LOCALIZATION_FEATURE);
+            /*
+             * Start of a temporary workaround to support both GraalVM 19.2.1 and 19.3.1 at the same time.
+             * TODO: Delete this workaround when Quarkus no longer supports GraalVM 19.2.1.
+             */
+            AssignableResultHandle locClass = overallCatch.createVariable(Class.class);
+            TryBlock workaroundTryBlock = overallCatch.tryBlock();
+            workaroundTryBlock.assign(locClass, workaroundTryBlock.loadClass(LOCALIZATION_FEATURE));
+            // The following line is required to throw an exception and make sure we load the 19.2.1 class when needed.
+            workaroundTryBlock.invokeVirtualMethod(
+                    ofMethod(Class.class, "getDeclaredMethod", Method.class, String.class, Class[].class), locClass,
+                    workaroundTryBlock.load("addBundleToCache"),
+                    workaroundTryBlock.marshalAsArray(Class.class, workaroundTryBlock.loadClass(String.class)));
```

Comment:
I tried to find a simpler expression to force the exception throw here, but I didn't find anything that worked. I'm sure this expression could be much shorter...

Reply:
I think it's just fine. You are just trying to make sure the method exists right? It should be perfectly OK and in any case it should be temporary

Reply:
Actually, I only need to check if the `com.oracle.svm.core.jdk.LocalizationFeature` class is available, I'm not really interested in the method itself. But invoking a simple `getName` from `LocalizationFeature` wasn't enough to force an exception throw. That's not really a problem since this code is only a copy of the original code a few lines below and it is also temporary, but that's still not pretty :)